src/Stratis.Bitcoin.Features.BlockStore/BlockStoreSignaled.cs

```diff
@@ -91,6 +85,12 @@ protected override void OnNextCore(ChainedHeaderBlock blockPair)
                 return;
             }
 
+            if (this.storeSettings.PruningEnabled)
```

Comment:
we should still announce blocks to peers when we are in pruned mode

Reply:
Are you sure? As this line was actually right at the top of the method. I had to move to blocks can be downloaded (but still not announced).